R/flatten.R

```diff
@@ -58,11 +58,17 @@ flatten <- function(tbl, groups, wide_cols, sep = "_") {
   rename <- colnames(out[[1]]) %in% wide_cols
   for (grp in names(out)) {
     group <- out[[grp]]
-    colnames(group)[rename] <- paste0(grp, sep, colnames(group)[rename])
+    colnames(group)[rename] <- if (any("x" == colnames(group))) {
```

Comment:
Just from logical point of view - maybe would be better `any(colnames(group)) == "x")` ?

Reply:
Done